enable/savage/svg/backends/wx/renderer.py

```diff
@@ -76,20 +76,32 @@ def createLinearGradientBrush(x1,y1,x2,y2, stops, spreadMethod='pad',
 
         stops = numpy.transpose(stops)
 
-        if len(stops) > 2:
-            warnings.warn("Wx only supports 2 gradient stops, but %d were specified" % len(stops))
-
         def convert_stop(stop):
             offset, red, green, blue, opacity = stop
             color = wx.Colour(red*255, green*255, blue*255, opacity*255)
             return offset, color
 
-        start_offset, start_color = convert_stop(stops[0])
-        end_offset, end_color = convert_stop(stops[1])
+        if wx.VERSION > (2,9):        
+            # wxPython 2.9+ supports a collection of stops
+            wx_stops = wx.GraphicsGradientStops()
+            for stop in stops:
+                offset, color = convert_stop(stop)
+                wx_stops.Add(color, offset)
+        
+            wx_renderer = wx.GraphicsRenderer.GetDefaultRenderer()
+            return wx_renderer.CreateLinearGradientBrush(x1, y1, x2, y2, wx_stops)
+        
+        else:        
+            if len(stops) > 2:
+                warnings.warn("Wx only supports 2 gradient stops, but %d were specified" % len(stops))
```

Comment:
Might be better to specifically say "Wx 2.8" or "Wx < 2.9" or something similar here so people know upgrading Wx may fix the problem.